ui/pages/SignIn.tsx

```diff
@@ -88,7 +88,11 @@ function SignIn() {
           <AlertWrapper
             severity="error"
             title="Error signin in"
-            message={`${String(error.status)} ${error.statusText}`}
+            message={`${
+              error.status == 401
+                ? "Incorrect username or password."
+                : String(error.status) + error.statusText
```

Comment:
Won't this look like: `401Unauthorized`?

Maybe: 
```suggestion
                : ` ${error.status):  ${error.statusText}`
```

Reply:
I made it like that ``${error.status} ${error.statusText}``